service/src/main/resources/db/changesets/20230530_avoid_postgres_specific_features.yaml

```diff
@@ -19,12 +35,12 @@ databaseChangeLog:
                   remarks: Deleting a study will cascade to delete its properties
               - column:
                   name: key
-                  type: text
+                  type: ${_text.type}
```

Comment:
This column is part of the unique constraint, should it be `id.type` instead?

Reply:
yes, if it is part of unique constraint, varchar(255) is OK too if key length if >50.